### PR TITLE
Fix: too many SQL variables on Delete

### PR DESCRIPTION
**Pull request by artemious7**
When deleting more than 999 records, an exception is thrown:

> Microsoft.WindowsAzure.MobileServices.SQLiteStore.SQLiteException: Error executing SQLite command: 'too many SQL variables'.

Deleting too many records can happen in the offline-sync scenario when PullAction is invoked on a table with more than 999 soft-deleted records, for instance when initial sync is executed.

### CODE REVIEW

**Comment by artemious7** [comment]
Will have to update the NuGet package version as well.

File: src/Microsoft.Azure.Mobile.Client.SQLiteStore/MobileServiceSQLiteStore.cs [hunk 1]
```diff
@@ -280,36 +280,46 @@ public override Task DeleteAsync(string tableName, IEnumerable<string> ids)
...
-                   try
-                   {
-                       this.ExecuteNonQueryInternal(sql, parameters);
-                   }
-                   finally
-                   {
-                       this.operationSemaphore.Release();
-                   }
-               });
+            this.ExecuteNonQueryInternal(sql, parameters);
```
Comment (abrull) [inline src/Microsoft.Azure.Mobile.Client.SQLiteStore/MobileServiceSQLiteStore.cs:322]: @artemious7 is there any reason to pass in the sql parameters after they're already in the sql string? I see you copied how it was done until now. Just wondering....

**Review by abrull — COMMENTED** [review]
@artemious7  Did you ever test this?

**Comment by artemious7** [comment]
@abrull yes
the package version (4.1.107) has yet to be updated in this PR.

**Comment by abrull** [comment]
@artemious7 Can you update the version? I don't have access to your branch

**Comment by artemious7** [comment]
@abrull please see https://github.com/Chorus/azure-mobile-apps-net-client/pull/11

**Pull request merged**